### DISCUSSION TITLE
"und" for underlines is messing up my workflow

### DISCUSSION THREAD

**Discussion by HeyUKonrad** [nvim-neorg/neorg#1346] [Syntax]
Hello guys!
First of all, I am very happy with neorg! I love the plugin and use it daily. For my studies but also for a DnD group I DM.
I have a small problem though. I write everything in German, and the shortcut for underlines "und" messes with my writing.
(und in German is and. So you could guess I use it quite a lot).
Is there any way to turn that off or change it?

**Comment by pysan3** [comment]
Those completions should be coming from friendly-snippets that you registered to luasnip. 

https://github.com/rafamadriz/friendly-snippets/blob/main/snippets/norg.json

AFAIK I don't know a way to only exclude a single keyword, but you may disable loading a single language so you might want to do that. 

https://github.com/rafamadriz/friendly-snippets/issues/271

Another option is to change the completion keybind so you don't trigger it when not needed. (Like `und<space>` shouldn't trigger the completion?)
